contracts/Token.sol

```diff
@@ -0,0 +1,27 @@
+pragma solidity ^0.5.0;
+
+import "openzeppelin-solidity/contracts/token/ERC20/ERC20.sol";
+import "openzeppelin-solidity/contracts/token/ERC20/ERC20Detailed.sol";
+
+contract Token is ERC20, ERC20Detailed {
+  uint40 startTime;
+  uint40 endTime;
+  constructor(uint40 _startTime, uint40 _endTime, uint256 initialSupply) ERC20Detailed("Gold", "GLD", 18) public {
+    _mint(msg.sender, initialSupply);
+    startTime = _startTime;
+    endTime = _endTime;
+  }
+
+   event Transfer(
```

Comment:
This event is already included in the ERC20 contracts. If you look at IERC20 (the interface for ERC20), you'll see it there. You will have to update your test to remove the underscores bc the event uses `from` `to` and `value`.